src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -7,7 +7,9 @@
     <RootNamespace>System.Xml</RootNamespace>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn),649</NoWarn>
-    <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
+    <FeatureCompiledXsl Condition="'$(TargetGroup)' != 'uap' AND '$(TargetGroup)' != 'uapaot'">true</FeatureCompiledXsl>
+    <DefineConstants Condition="'$(FeatureCompiledXsl)' == 'true'">$(DefineConstants);FEATURE_COMPILED_XSL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);FEATURE_SERIALIZATION_UAPAOT</DefineConstants>
```

Comment:
I suggested @krwq to `FEATURE_SERIALIZATION_UAPAOT` at the beginning because the serialization code that was under `uapaot` was the code that knows how to use the serializer pre-generated by UWP toolchain. As UWP toolchain is used on UAPAOT only, so I suggested that name.

But I agreed with you that we shouldn't put `uap` or `uapaot` in the name. So I thought about using names like `FEATURE_USE_PREGENERATED_SERIALIZER`. The concern about use this name was that it might cause confusion between using the pre-generated serializer by UWP toolchain and using the pre-generated serializer by SGen.exe (not toolchain).

I think `FEATURE_USE_AOT_SERIALIZER` would fit here the best, which means using the ahead of time serializer, i.e. generated by UWP toolchain.